### PR TITLE
fix dashboard typos

### LINKED ISSUE #4421: bug: Some typos in grafana dashboard
*Description*:
>What issue is being seen? Describe what should be happening instead of
the bug, for example: Envoy should not crash, the expected value isn't
returned, etc.

*Repro steps*:
> Include sample requests, environment, etc. All data and inputs
required to reproduce the bug.

Upstream Bytes Tx/Second uses `RX` metric by mistake.
image

Allocated memroy uses `SUM` by mistake (should be lower `sum`), causes dashboard crash when being edited.
image

>**Note**: If there are privacy concerns, sanitize the data prior to
sharing.

*Environment*:
>Include the environment like gateway version, envoy version and so on.

*Logs*:
>Include the access logs and the Envoy logs.

### PR DESCRIPTION
**Pull request by haorenfsa**
**What type of PR is this?**

fix: some typos in Grafana dashboard.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #4421

### CODE REVIEW

**Comment by zirain** [comment]
IIRC, that json generated from jsonnet, can you update it and then run `make -k gen-check`?

cc @shawnh2

File: charts/gateway-addons-helm/dashboards/envoy-proxy-global.json [hunk 1]
```diff
@@ -1849,7 +1849,7 @@
             "uid": "${datasource}"
           },
           "editorMode": "builder",
-          "expr": "sum by(namespace) (rate(envoy_cluster_upstream_cx_rx_bytes_total{namespace=~\"$Namespace\"}[5m]))",
+          "expr": "sum by(namespace) (rate(envoy_cluster_upstream_cx_tx_bytes_total{namespace=~\"$Namespace\"}[5m]))",
```
Comment (shawnh2) [inline charts/gateway-addons-helm/dashboards/envoy-proxy-global.json:1852]: good catch!

**Review by shawnh2 — DISMISSED** [review]

**Comment by shawnh2** [comment]
> IIRC, that json generated from jsonnet, can you update it and then run `make -k gen-check`?
> 
> 
> 
> cc @shawnh2 

not really, these json are legacy dashboards, not generated from jsonnet.

**Comment by haorenfsa** [comment]
> please run make -k gen-check to make the CI pass.
@shawnh2 done

**Review by shawnh2 — APPROVED** [review]
LGTM! Thanks

**Review by zirain — APPROVED** [review]

**Pull request merged**